src/components/screens/AddAssetScreen/AddAssetBottomSheetContent.tsx

```diff
@@ -0,0 +1,137 @@
+import { AssetIcon } from "components/AssetIcon";
+import { FormattedSearchAssetRecord } from "components/screens/AddAssetScreen/types";
+import Avatar from "components/sds/Avatar";
+import { Badge } from "components/sds/Badge";
+import { Button, IconPosition } from "components/sds/Button";
+import Icon from "components/sds/Icon";
+import { Text } from "components/sds/Typography";
+import { ActiveAccount } from "ducks/auth";
+import { pxValue } from "helpers/dimensions";
+import useAppTranslation from "hooks/useAppTranslation";
+import useColors from "hooks/useColors";
+import React from "react";
+import { View } from "react-native";
+
+type AddAssetBottomSheetContentProps = {
+  asset: FormattedSearchAssetRecord | null;
+  account: ActiveAccount | null;
+  onCancel: () => void;
+  onAddAsset: () => void;
+  isAddingAsset: boolean;
+};
+
+const AddAssetBottomSheetContent: React.FC<AddAssetBottomSheetContentProps> = ({
+  asset,
+  account,
+  onCancel,
+  onAddAsset,
+  isAddingAsset,
+}) => {
+  const { themeColors } = useColors();
+  const { t } = useAppTranslation();
+
+  if (!asset) {
+    return null;
+  }
+
+  return (
+    <View className="flex-1 justify-center items-center mt-2">
+      <AssetIcon
+        token={{
+          ...(asset.isNative
+            ? { type: "native" }
+            : {
+                type: "credit_alphanum4",
+              }),
+          code: asset.assetCode,
+          issuer: {
+            key: asset.issuer,
+          },
+        }}
```

Comment:
looks like this code is duplicated on `AddAssetBottomSheetContent` and `AssetItem` components. Could we create a helper for it?